tests/test_kernels/test_nonstationary.py

```diff
@@ -101,7 +96,9 @@ def test_initialization(self, fields: dict, dim: int) -> None:
         # Check meta leaves
         meta = kernel._pytree__meta
         assert not any(f in meta for f in self.static_fields)
-        assert list(meta.keys()) == sorted(set(fields) - set(self.static_fields))
+        assert sorted(list(meta.keys())) == sorted(
```

Comment:
Note that this is a bug fix!

Reply:
Its triggered as I have a bias_variance param which sits near the start of the alphabet